internal/beater/beater.go

```diff
@@ -387,7 +387,7 @@ func (s *Runner) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	batchProcessor := modelprocessor.Chained{
+	batchProcessor := srvmodelprocessor.NewTracer("beater.Processor", modelprocessor.Chained{
```

Comment:
`beater.ProcessBatch`?
```suggestion
	batchProcessor := srvmodelprocessor.NewTracer("beater.ProcessBatch", modelprocessor.Chained{
```